x-pack/test/apm_api_integration/basic/tests/annotations.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { JsonObject } from 'src/plugins/kibana_utils/common';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function annotationApiTests({ getService }: FtrProviderContext) {
```

Comment:
Should we seek truce with the linter and get rid of default export?

Reply:
I don't think that's compatible with the way tests are loaded (you pass absolute paths)

Reply:
I've seen similar linter ignore comments throughout `~/x-pack/test`; it seems like an institutional problem. Perhaps instead we should disable `no-default-export` for this directory, or certain subdirs.

Reply:
Yeah, looks like some of the test dirs already have an exception from what I can tell from `.eslintrc.js`. I'll punt on it though if you don't mind given the amount of teams already involved in this PR 😅